src/npctalk.cpp

```diff
@@ -1071,6 +1072,217 @@ std::string dialogue::dynamic_line( const std::string &topic ) const
+
+    } else if( topic == "TALK_RANCH_STOCKS_BANDAGES" ) {
+             return talk_function::bulk_trade_inquire(p, "bandages");
+
+    } else if( topic == "TALK_RANCH_DOCTOR" ) {
+             return _("I'm sorry, I don't have time to see you at the moment.");
+
+    } else if( topic == "TALK_RANCH_SCRAPPER" ) {
+             return _("Don't mind me.");
+
+    } else if( topic == "TALK_RANCH_SCRAPPER_JOB" ) {
+             return _("I chop up useless vehicles for spare parts and raw materials.  If we can't use a "
+                      "vehicle immediately we haul it into the ring we are building to surround the outpost... "
+                      "it provides a measure of defense in the event that we get attacked.");
+
+    } else if( topic == "TALK_RANCH_SCRAPPER_HIRE" ) {
+             return _("I don't personally, the teams we send out to recover the vehicles usually need a hand "
+                      "but can be hard to catch since they spend most of their time outside the outpost.");
+
+    } else if( topic == "TALK_RANCH_SCAVENGER_1" ) {
+             return _("Welcome to the junk shop...");
+
+    } else if( topic == "TALK_RANCH_SCAVENGER_1_JOB" ) {
+             return _("I organize scavenging runs to bring in supplies that we can't produce ourselves.  I "
+                      "try and provide incentives to get migrants to join one of the teams... its dangerous "
+                      "work but keeps our outpost alive.  Selling anything we can't use helps keep us afloat "
+                      "with the traders.  If you wanted to drop off a companion or two to assist in one of "
+                      "the runs, I'd appreciate it.");
+
+    } else if( topic == "TALK_RANCH_SCAVENGER_1_HIRE" ) {
+             return _("Are you interested in the scavenging runs or one of the other tasks that I might have for you?");
+
+    } else if( topic == "TALK_RANCH_BARKEEP" ) {
+             return _("Want a drink?");
+
+    } else if( topic == "TALK_RANCH_BARKEEP_JOB" ) {
+             return _("If it isn't obvious, I oversee the bar here.  The scavengers bring in old world alcohol that we "
+                      "sell for special occasions.  For most that come through here though, the drinks we brew "
+                      "ourselves are the only thing they can afford.");
+
+    } else if( topic == "TALK_RANCH_BARKEEP_INFORMATION" ) {
+             return _("We have a policy of keeping information to ourselves.  Ask the patrons if you want to hear "
+                      "rumors or news.");
+
+    } else if( topic == "TALK_RANCH_BARKEEP_TAP" ) {
+             return _("Our selection is a bit limited at the moment.");
+
+    } else if( topic == "TALK_RANCH_BARBER" ) {
+             return _("Can I interest you in a trim?");
+
+    } else if( topic == "TALK_RANCH_BARBER_JOB" ) {
+             return _("What?  I'm a barber... I cut hair.  There's demand for cheap cuts and a shave out here.");
```

Comment:
I've had a laughing fit over this one, imagining my character hoisting her hairy leg up onto the counter and saying, "Give me a shave, my good man."